packages/cma-client-node/__tests__/item.test.ts

```diff
@@ -276,4 +277,33 @@ describe('item', () => {
     expect(updatedContent[0].attributes.text).toEqual('Updated Foo');
     expect(updatedContent[1].attributes.text).toEqual('Updated Bar');
   });
+
+  it.concurrent('create with explicit ID', async () => {
+    const client = await generateNewCmaClient();
+
+    const newId = generateId();
+
+    const itemType = await client.itemTypes.create({
+      name: 'Article',
+      api_key: 'article',
+    });
+
+    await client.fields.create(itemType.id, {
+      label: 'Title',
+      field_type: 'string',
+      api_key: 'title',
+    });
+
+    const item = await client.items.create({
+      id: newId,
+      title: 'My first blog post',
+      item_type: itemType,
+    });
+
+    await client.items.bulkPublish({
+      items: [item],
+    });
```

Comment:
The client.items.bulkPublish here is not useful, is not part of what we want to test.